.azure-pipelines/testscripts_analyse/analyse_testscripts.py

```diff
@@ -145,10 +153,10 @@ def get_PRChecker_scripts():
             else:
                 topology_type_pr_test_scripts[topology_type] = set(value)
 
-    return topology_type_pr_test_scripts
+    return topology_type_pr_test_scripts, topology_type_pr_test_skip_scripts
 
 
-def check_PRChecker_coverd(test_scripts, topology_type_pr_test_scripts):
+def check_PRChecker_coverd(test_scripts, topology_type_pr_test_scripts, topology_type_pr_test_skip_scripts):
```

Comment:
Nit: I guess we could change the name to `expand_test_scripts` as it takes the current list and expands `any` type to all of the types in `PR_TOPOLOGY_TYPE`. And the function then returns `expanded_test_scripts` which is easier to understand. I feel `check_` functions usually represent returning status / boolean values.

Reply:
Changed